apps/meteor/client/hooks/useLicense.ts

```diff
@@ -0,0 +1,32 @@
+import { CloudRegistrationStatus, ILicense } from '@rocket.chat/core-typings';
+import { useEndpoint } from '@rocket.chat/ui-contexts';
+import { useQuery } from '@tanstack/react-query';
+
+export const useLicense = (): {
+	licenses: ILicense[] | undefined;
+	registrationStatus: CloudRegistrationStatus | undefined;
+	isError: boolean;
+	isLoading: boolean;
+	isSuccess: boolean;
+} => {
+	const getRegistrationStatus = useEndpoint('GET', '/v1/cloud.registrationStatus');
+	const getLicenses = useEndpoint('GET', '/v1/licenses.get');
+
+	const { data: registrationStatusData } = useQuery(['licenses', 'getRegistrationStatus'], () => getRegistrationStatus(), {
```

Comment:
`registrationStatus` and `licenses` are somewhat related, but at the same time independent from each other. Can you change this key? Thanks.

Reply:
Done!